Dependencies/xr/Source/ARCore/XR.cpp

```diff
@@ -456,6 +481,10 @@ namespace xr
 
                 // Draw the quad
                 glDrawArrays(GL_TRIANGLE_STRIP, 0, VERTEX_COUNT);
+
+                // Present to the screen
+                eglSwapBuffers(eglGetCurrentDisplay(), eglGetCurrentSurface(EGL_DRAW));
```

Comment:
Can you add a note that this breaks the non-react-native implementation, so that we can easily find and swap it out when working locally?